docs/bot-usage.md

```diff
@@ -145,9 +145,9 @@ It is recommended to use version control to keep track of changes to your strate
 
 This parameter will allow you to load your custom strategy class.
 Per default without `--strategy` or `-s` the bot will load the
```

Comment:
"Per default without `--strategy` or `-s` the bot will load the" -- I assume this whole sentence is wrong... (the bot does not load SampleStrategy without `--strategy` or `-s` options)

Reply:
mmh, true enough :D